scripts/create-rbac-users.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Create rbac users
+
+if [ -e "/startup/rbac-users.json" ]; then
+  while read rbacSettings
+  do
+    rbacBucketName=$(echo $rbacSettings | jq -r '.["bucket"]')
+    rbacName=$(echo $rbacSettings | jq -r '.["rbacName"]')
+    rbacUsername=$(echo $rbacSettings | jq -r '.["rbacUsername"]')
+    rbacPassword=$(echo $rbacSettings | jq -r '.["rbacPassword"]')
+    rbacRoles=$(echo $rbacSettings | jq -r '.roles | join(",")')
+
+    echo "Bucket Name = $rbacBucketName"
+
+    if [[ ! -z $rbacBucketName ]]; then
```

Comment:
My only remaining concern is the presence of bucketName on the JSON doc and validating that the folder exists. I see two concerns. First, you can make a bucket in buckets.json without having a bucket folder that defines indexes/models. Second, it's theoretically possible for a user to not have any specific bucket association and be purely admin in nature.

Do you have a specific use case in mind for this property?

Reply:
I see your point. I'll remove the bucket property from the JSON and the rbacBucketName variable

Reply:
I've made the requested changes in this [commit](https://github.com/brantburnett/couchbasefakeit/pull/25/commits/b5f67259c004ee7f9db64316639ef0b4c35bb9e3)